### DISCUSSION TITLE
Matrix room?

### DISCUSSION THREAD

**Discussion by poperigby** [organicmaps/organicmaps#556] [Q&A] [answered]
It would be neat if OrganicMaps had a [Matrix](https://matrix.org) room bridged with the Telegram room. Is this something you would be willing to do? It would just required creating a Matrix room and bridging it with the Telegram room using a bot. You could either selfhost it or use [this](https://t2bot.io/telegram/) one.

**Comment by rtsisyk** [comment]
We tried Matrix <-> TG bot at the beginning, and it was really annoying. TG doesn't allow Matrix Bot to spoof usernames, so all messages come from the bot name.

**Reply by poperigby** [reply]
I see. That's frustrating.

**Reply by rtsisyk** [reply]
We have Matrix now. Can we mark this question as answered?

**Comment by biodranik** [comment, marked as answer]
Here is our Matrix room: https://matrix.to/#/#organicmaps:matrix.org
